app/cdap/components/RecipeList/styles.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+export const RecipeTableDiv = styled(({ ...props }) => <div {...props} />)`
+    width: 100%;
+    box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.2), 0px 2px 2px rgba(0, 0, 0, 0.14), 0px 1px 5px rgba(0, 0, 0, 0.12);
+    background-color: #FFFFFF;
+    border-radius: 4px;
+
+    .grid.grid-container {
```

Comment:
I don't know that we want to have deeply nested class-based styling within the styled-components block? How much work would it be to individually style these components?

Reply:
As I am using common classes for table styles, making it to individual styles components requires restructuring these components and re define the classes. So would like to take it as a separate task.